kornia/augmentation/augmentation.py

```diff
@@ -2295,17 +2357,31 @@ def __init__(
 
     def __repr__(self) -> str:
         return self.__class__.__name__ + f"({super().__repr__()})"
+    
+    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor]:
+        input_size = torch.tensor(batch_shape[-2:], dtype=torch.long).expand(batch_shape[0], -1)
+        return dict(input_size=input_size)
 
-    # TODO(jian): make somehhow that if it's not declared, returns the identity
-    # TODO: compute the proper the proper transformation
+    # TODO: It is incorrect to return identity
+    # TODO: Having a resampled version with ``warp_affine``
     def compute_transformation(self, image: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
         return self.identity_matrix(image)
 
     def apply_transform(
-        self, image: torch.Tensor, params: Dict[str, torch.Tensor], transform: Optional[torch.Tensor] = None
+        self, input: torch.Tensor, params: Dict[str, torch.Tensor], transform: Optional[torch.Tensor] = None
```

Comment:
https://github.com/kornia/kornia/issues/1276#issuecomment-918933837 ;)

Reply:
the function signature will need to align with the parent class. If you gonna work on it, I would suggest to do it at once, not for one or two functions for consistency. Also, please keep in mind about the deprecation warnings.